index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <nav class="toolbar">
+            <div class="my-logo primary-color hover-underline">My Logo</div>
+            <div class=" collapsed_menu primary-color">&#9776;</div>
+        </nav>
+        <!--END OF TOOLBAR SECTION-->
+
+        <!--HEADLINE SECTION-->
+        <section>
+            <div>
+                <div><img class="ellipse11" src="images/Ellipse11.jpg"></div>
+                <div class="ellipse_5"><img  src="images/Ellipse 5.jpg"></div>
+                <div class="rectangle_7"><img  src="images/Rectangle 7.jpg"></div>
+                <div class="rectangle_4"><img src="images/Rectangle 4.jpg"> </div>
+                <div class="ellipse_3"><img  src="images/Ellipse 3.jpg"></div>
+            </div>
```

Comment:
It is considered best practice to style the background of a section using CSS as opposed to using img tags in the HTML👍. Kindly style this section using the `background-image:` CSS property🙏. You can select all the shapes in the figma template and export it as an image file